CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
    * ADDED: Initial implementation of turn lanes.
   **Bug Fix**
    * CHANGED: Fix destination only penalty for A* and time dependent cases.
+   * CHANGED: Use the distance from GetOffsetForHeading, based on road classification and use, within tangent_angle function.
```

Comment:
this seems oddly worded, my brain cant parse the word use twice in here or something. maybe you meant:

    Use the distance from GetOffsetForHeading, based on road classification, within tangent_angle function.

Reply:
means road classification and road use (e.g. ramp, turn channel, etc.)